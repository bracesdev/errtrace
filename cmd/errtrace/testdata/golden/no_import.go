//go:build ignore

package foo

type myError struct{}

func (*myError) Error() string {
	return "sadness"
}

func Try() error {
	return &myError{}
}
