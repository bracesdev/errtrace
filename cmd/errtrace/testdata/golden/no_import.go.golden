//go:build ignore

package foo; import "braces.dev/errtrace"

type myError struct{}

func (*myError) Error() string {
	return "sadness"
}

func Try() error {
	return errtrace.Wrap(&myError{})
}
